cmake/OpenCVModule.cmake

```diff
@@ -1175,7 +1175,7 @@ function(ocv_add_perf_tests)
         set_target_properties(${the_target} PROPERTIES FOLDER "tests performance")
       endif()
 
-      if(WINRT)
+      if(WINRT AND BUILD_TESTS)
         # removing APPCONTAINER from tests to run from console
         # look for detailed description inside of ocv_create_module macro above
         add_custom_command(TARGET "opencv_perf_${name}"
```

Comment:
Performance tests are under `BUILD_PERF_TESTS` guard (line 1125).
Perhaps we need to turn off both `BUILD_TESTS` / `BUILD_PERF_TESTS` flags in vcpkg.

Reply:
Thanks for the review! 
I reverted to just `if(WINRT)` for this instance.